### ISSUE TITLE
Rich-text-editor's integration with tributeJS no longer working

### ISSUE THREAD

**Issue by rashisf** [syncfusion/ej2-angular-ui-components#220]
I have upgraded my text editor from version `19.2.62` to latest `19.4.47`.

I am using `tributejs` library for mentioning users & had successfully integrated with the text editor.
However after version upgrade, it has stopped working.

When I type '@' , the list of users comes up. After selecting one user & pressing enter, the editor does not update the name of user in the user. It comes out empty & hence is not reflected in the box.

I am using `selectTemplate` function of `tributejs` to return the template of the user data selected but is not reflected. It used to work in version `19.2.62`.

I need this working with the latest version.

**Comment by BuvanaSathasivam** [comment]
Hi Rashisf,

Greetings from Syncfusion support.

We have checked your problem. But, we were unable to reproduce your reported problem. We have checked with the latest source (19.4.47) with tributeJs and the selected value is reflected in the Rich Text Editor content after enter key pressed. Can you please find the below code?

**Sample:**  https://stackblitz.com/edit/angular-1efpad?file=app.component.ts 

Could you please share the below information,
-	Share your code or modify the above sample to address your issue.
-	Share the video to better understand your issue.

Regards,
Buvana S

**Comment by thangavele** [comment]
Hi Rashisf,

We are closing this issue since there is no recent activity. Feel free to reopen this and update your comments for the reinvestigation.

Regards
Thangavel E

**Comment by rashisf** [comment]
Hi @BuvanaSathasivam 

I just checked the [stackblitz](https://stackblitz.com/edit/angular-1efpad?file=app.component.ts) that you shared. I was able to solve my issue as I need to additionally add the following snippet of code to my code. 

 ` public actionBeginEvent(args: any): void {
      if (args.requestType === 'EnterAction') {
        args.cancel = true;
      }
    }`
    
 However, before the version upgrade it was never really needed. But this resolves the issue anyway.

Thank you for resolving my query promptly.

Regards,
Rashi Sharma
